guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/registering-slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+::: warning
+Make sure that you're on discord.js version 13.3.0 or above before using autocomplete.
+:::
+
+## Preparing commands
+
+To use autocomplete with your command options, you have to tell Discord to enable it.
+How to do this is dependent on how you create your slash commands.
+
+### Slash command builder
+
+The slash command builder does not yet support using autocomplete.
+
+### Custom system
+
+For other custom systems or entering it raw, your resulting code should look something like this. To enable autocomplete, set it to true in the option.
+
+```js {10}
+const commandData = {
+	name: 'autocomplete',
+	description: 'test command to show how autocomplete should be setup',
+	defaultPermission: true,
+	options: [
+		{
+			name: 'name',
+			description: 'Name of something',
+			type: 3, // String
+			autocomplete: true,
+		},
+	],
+};
+```
+
+## Receiving
+
+To receive an <DocsLink path="class/AutocompleteInteraction" />, you can listen to the `interactionCreate` event and use the <DocsLink path="class/Interaction?scrollTo=isAutocomplete" /> method to make sure you only receive autocomplete interactions:
+
+```js {2}
+client.on('interactionCreate', interaction => {
+	if (!interaction.isAutocomplete()) return;
+	console.log(interaction);
+});
+```
+
+## Responding
+
+The <DocsLink path="class/AutocompleteInteraction" /> class provides the <DocsLink path="class/AutocompleteInteraction?scrollTo=respond" /> method for responding to the interaction.
+
+### Respond with search
+Using `AutocompleteInteraction#respond()` you submit an array of ApplicationCommandOptionChoice objects. Passing an empty array will show "No options match your search" in Discord.
```

Comment:
```suggestion
Using `AutocompleteInteraction#respond()` you submit an array of ApplicationCommandOptionChoice objects. Passing an empty array will display "No options match your search" within the Discord client.
```
This needs to be rephrased to feel more organic and professional, `in Discord` again sounds how a child would describe it.